CHANGELOG.md

```diff
@@ -11,6 +11,12 @@ the code was deployed.
 
 ## [Unreleased]
 
+## [XX.X.X] - 2024-XX-XX
```

Comment:
no need for this, `## [Unreleased]` is sufficient as a header

Reply:
Fixed!